applications/HDF5Application/python_scripts/hdf5_io.py

```diff
@@ -283,3 +283,31 @@ def _get_current_file_name(self):
         fmt = "{:." + str(self._time_tag_precision) + "f}"
         time_tag = "-" + fmt.format(self._model_part.ProcessInfo[KratosMultiphysics.TIME])
         return self._time_step_file_name + time_tag + ".h5"
+
+class InitializationFromInputProcess(KratosMultiphysics.Process):
+    """A process to initialize a simulation using an HDF5 output file."""
+
+    def __init__(self, model_part, hdf5_file_factory, settings):
+        KratosMultiphysics.Process.__init__(self)
+        default_settings = KratosMultiphysics.Parameters(r'''{
+            "file_name": ""
+        }''')
+        settings.ValidateAndAssignDefaults(default_settings)
+        file_name_string = settings["file_name"].GetString()
+        if file_name_string == "":
+            raise Exception("Please specify the HDF5 input file name as the \"file_name\" parameter (string).")
+        else:
+            self._file_name = file_name_string
+
+        self._model_part = model_part
```

Comment:
Would it be ok to have this as a setting in the json? So that we can use this process to initialize a sub-model and use other initialization processes to initialize other sub-model parts?